roles/cinder-data/tasks/monitoring.yml

```diff
@@ -7,3 +7,9 @@
 - name: iscsid process check
   sensu_process_check: service=iscsid
   notify: restart sensu-client
+
+- name: cinder volume group metrics
+  sensu_metrics_check: name=cinder-vg-metrics plugin=metrics-volgroup.sh
+                       args='-v cinder-volumes --scheme {{ monitoring.graphite.host_prefix }}'
+  notify: restart sensu-client
+  when: cinder.fixed_key is not defined or (cinder.fixed_key is defined and "compute" not in group_names)
```

Comment:
you're crazy @j2sol 

it will run if fixed_key is /not/ defined,  or if fixed_key is defined but is not a compute node.

in fact it's a duplicate of the check that you wrote to determine if we should lay down the `lvm` volume in the first place - https://github.com/blueboxgroup/ursula/blob/3c3015ed77f7570c44e01ba70315172520c62f02/roles/cinder-data/tasks/main.yml#L28-L29